plugins/status/plugin.go

```diff
@@ -258,6 +261,24 @@ func (p *Plugin) register(r prom.Registerer, cs ...prom.Collector) {
 	}
 }
 
+func (p *Plugin) registerAll() {
+	if p.manager.PrometheusRegister() != nil {
+		p.register(p.manager.PrometheusRegister(), allCollectors...)
+	}
+}
+
+func (p *Plugin) unregister(r prom.Registerer, cs ...prom.Collector) {
+	for _, c := range cs {
+		r.Unregister(c)
+	}
+}
```

Comment:
[comment]: This is a clever trick for getting some API symmetry between `register`/`unregister`. :smile: